enhancements/installer/vsphere-ipi-zonal-external-lb.md

```diff
@@ -0,0 +1,201 @@
+---
+title: vsphere-ipi-zonal-external-lb
+authors:
+  - "@jcpowermac"
+reviewers:
+  - "@rvanderp3"
+  - "@bostrt"
+approvers:
+  - "@rvanderp3"
+  - "@patrickdillon"
+api-approvers:
+  - "@JoelSpeed"
+  - "@deads2k"
+creation-date: 2022-11-02
+last-updated: 2022-11-03
+status: implementable
+see-also:
+  - "/enhancements/installer/vsphere-ipi-zonal.md"
+replaces:
+  - None
+superseded-by:
+  - None
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-9652
+
+---
+
+# Support External Load Balancer in vSphere Zonal IPI
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The goal of this enhancement is to provide the ability to install vSphere IPI zonal on separate L2 segments without the static pods (keepalived, haproxy, coredns). 
+As of this writing the only way to accomplish this is to allow an external load balancer.
+
+## Motivation
+
+Users of OpenShift would like the ability to deploy control plane and compute nodes in more than one L2 segment to increase reliability and ease of network management.
+
+### Goals
+
+- Support external load balancer in vSphere IPI zonal
+- Support multiple subnets via the external load balancer
+- Support for no static pod deployment (keepalived, haproxy, coredns)
+
+### Non-Goals
+
+- Provide a mechanism to configure a external load balancer from the installer or machine-api-operator 
+
+## Proposal
+
+We propose the use of small L2 segments (e.g. /28). The customer's
+load balancer will be pre-configured with the entire subnet as end points
+for the required ports. Those L2 segments will be a associated with vSphere port groups that will be used in the `FailureDomains` and provisioning process.  The external load balancer will discover available endpoints and use those accordingly.
+
+- Modification to the Infrastructure status object to include new LoadBalancerType
+- Modification to the platform spec object to include new LoadBalancerType
+- Modification to the machine-config-operator to use new LoadBalancerType 
+
+### Workflow Description
+
+### api
+
+#### Infrastructure Status
```

Comment:
what about existence / non-existence of VIPs - why it cannot be the signal? then it will be aligned with upi/ipi networking parts

Reply:
and, the installer can set the lodbalancer type as the derivative of those values, without letting the user specifying it